src/core/link-to-dfn.js

```diff
@@ -237,11 +237,11 @@ function findExplicitExternalLinks() {
   );
   return [...links]
     .filter(el => {
+      // ignore empties
+      if (el.textContent === "") return false;
```

Comment:
Why removing `.trim()`?

Reply:
The convention is `<a data-cite="thing"></a>` to autofill a thing, but I guess to be safe it should trim.

Reply:
No strong opinion but a formatter may insert a newline between tag opening and close.

Reply:
Yeah, I was thinking about that too... I don't think they do for `<a>` (at least tidy doesn't) because it would create an empty link with a space in it.

Reply:
AFAIK the HTML whitespace rule won't render a space if the element follows or followed by whitespaces.

Reply:
ah, ok. I wasn't aware. Browsers are fun 😍